### PR TITLE
add faq

### PR DESCRIPTION
**Pull request by edsu7**
Adding FAQ to address questions from heldpesk

### CODE REVIEW

**Review by justincorrigible — DISMISSED** [review]
LGTM 👍 

it may be worth adding a link to this section in the new "retired" page, and somewhere visible in the docs home page? @joneubank

File: docs/data-access/icgc-25k-data.md [hunk 1]
```diff
@@ -92,3 +92,25 @@ Please see ICGC ARGO's [publication policy](https://www.icgc-argo.org/page/77/e3
...
+PCAWG/pcawg_dkfz_caller                      PCAWG/pilot50-mosaic                         PCAWG/pilot50_calls                          PCAWG/quality_control_info                   PCAWG/reference_data                         PCAWG/retrotransposition                     PCAWG/rnaseq_aligned_bams                    PCAWG/sanger_calls
+PCAWG/sequencing_metadata                    PCAWG/smufin_indel_calls                     PCAWG/subclonal_reconstruction               PCAWG/terminology_and_standard_colours       PCAWG/thesaurus_snv                          PCAWG/transcriptome                          PCAWG/unaligned_bams                         PCAWG/validation_bams
+PCAWG/wgs_aligned_bams 
+```
+
+### 2. Connecting through SFTP gives the following error `no matching host key type found. Their offer: ssh-rsa`
+
+This will depends on system but ensure your `~/.ssh/.config` has the following content:
```
Comment (joneubank) [inline docs/data-access/icgc-25k-data.md:111]: All connections are over SFTP. Do you specifically mean a specific `sftp` CLI?

File: docs/data-access/icgc-25k-data.md [hunk 2]
```diff
@@ -92,3 +92,25 @@ Please see ICGC ARGO's [publication policy](https://www.icgc-argo.org/page/77/e3
 ## Questions and Concerns
 
 If you have any further questions or require additional information please contact the [helpdesk](https://platform.icgc-argo.org/contact).
+
+## Frequently asked questions
+
+### 1. I cannot find anything inside the **Example folder**
```
Comment (joneubank) [inline docs/data-access/icgc-25k-data.md:98]: This also addresses the syntax for a specific SFTP tool. Can we update this text to include a reference to the tool who's interface is being clarified?

"I am using the `sftp` CLI and cannot find anything inside a directory" or something

**Review by joneubank — CHANGES_REQUESTED** [review]
You are providing instructions for a specific client without clearly indicating what client this information is pertinent to.

File: docs/data-access/icgc-25k-data.md (hunk 1 above)
Comment (edsu7) [reply docs/data-access/icgc-25k-data.md]: Updated to [Using CLI SFTP, I cannot find anything inside the Example folder](https://github.com/icgc-argo/argo-docs/blob/feat/faq/docs/data-access/icgc-25k-data.md#1-using-cli-sftp-i-cannot-find-anything-inside-the-example-folder)

File: docs/data-access/icgc-25k-data.md (hunk 2 above)
Comment (edsu7) [reply docs/data-access/icgc-25k-data.md]: Updated to [Connecting through CLI SFTP gives the following error no matching host key type found. Their offer: ssh-rsa](https://github.com/icgc-argo/argo-docs/blob/feat/faq/docs/data-access/icgc-25k-data.md#2-connecting-through-cli-sftp-gives-the-following-error-no-matching-host-key-type-found-their-offer-ssh-rsa)

**Review by joneubank — APPROVED** [review]

**Pull request merged**